modules/flowable-engine/src/main/java/org/flowable/engine/impl/bpmn/helper/ErrorPropagation.java

```diff
@@ -160,7 +160,7 @@ protected static void executeCatch(Map<String, List<Event>> eventMap, DelegateEx
                             String refActivityId = refId.substring(0, refId.indexOf('#'));
                             String refProcessDefinitionId = refId.substring(refId.indexOf('#') + 1);
                             if (parentExecution.getProcessDefinitionId().equals(refProcessDefinitionId) &&
-                                            currentContainer.getFlowElement(refActivityId) != null) {
+                                            currentContainer.getFlowElement(refActivityId,false) != null) {
```

Comment:
```suggestion
                                            currentContainer.getFlowElement(refActivityId, false) != null) {
```
Project standards is to include a space between parameters